ChatBot/README.md

```diff
@@ -0,0 +1,14 @@
+Packages needed to be installed: 
```

Comment:
It's quite interesting; just a little change is required: change the name of the markdown file to something else, as it might get a bit confusing with the README.md of the main repo

Reply:
I've changed the README.md filename to Instructions_to_use_it.md, please check.